FE/src/hooks/pages/Interview/useWindowSize.ts

```diff
@@ -0,0 +1,33 @@
+import { useState, useEffect } from 'react';
+
+const useWindowSize = () => {
+  const [windowSize, setWindowSize] = useState({
+    width: window.innerWidth,
+    height: window.innerHeight,
+  });
+
+  useEffect(() => {
+    let timeoutId: number | undefined;
+
+    const handleResize = () => {
+      window.clearTimeout(timeoutId);
+      timeoutId = window.setTimeout(() => {
+        setWindowSize({
+          width: window.innerWidth,
+          height: window.innerHeight,
+        });
+      }, 250);
+    };
+
+    window.addEventListener('resize', handleResize);
+
+    return () => {
+      window.removeEventListener('resize', handleResize);
+      if (timeoutId !== undefined) window.clearTimeout(timeoutId);
+    };
+  }, []);
```

Comment:
[p-4]
```suggestion
  useEffect(() => {
    let timeoutId: number;

    const handleResize = () => {
      window.clearTimeout(timeoutId);
      timeoutId = window.setTimeout(() => {
        setWindowSize({
          width: window.innerWidth,
          height: window.innerHeight,
        });
      }, 250);
    };

    window.addEventListener('resize', handleResize);

    return () => {
      window.removeEventListener('resize', handleResize);
      window.clearTimeout(timeoutId);
    };
  }, []);
```
window.clearTimeout으로 넘기는 인자 타입이 number|undefiend이라서 `if (timeoutId !== undefined) window.clearTimeout(timeoutId);`을 확인하지 않아도 됩니다!

Reply:
네 확인했습니다!! 감사합니다

Reply:
c6bcdf70cb729721e0c8a21b3ea6782690342571
반영완료!